src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,33 @@
+import PropTypes from 'prop-types';
+import './TimeStamp';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = () => {
+function ChatEntry({ sender, body, timeStamp, liked, onLikeToggle, isLocal }) {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={`chat-entry ${isLocal ? 'local' : 'remote'} ${liked ? 'liked' : ''}`}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <TimeStamp time={timeStamp}/>
+        <button
+          className="like"
+          onClick={onLikeToggle}
```

Comment:
In `ChatLog.jsx`, we should just pass the function `onLikeToggle` and also `id` and invoke it here in the component where it's needed. 

```js
const onLikeClick = () => {
  onLikeToggle(id);
};

<button
  className="like"
  onClick={onLikeClick}>
  // rest of your logic
</button>
```